zerver/views/webhooks/pingdom.py

```diff
@@ -0,0 +1,65 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from zerver.models import get_client
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+import ujson
+
+
+PINGDOM_SUBJECT_TEMPLATE = 'Your Pingdom {name} descries something important.'
+PINGDOM_MESSAGE_TEMPLATE = 'Service {service_url} changed it\'s status from {previous_state} to {current_state}.'
+PINGDOM_MESSAGE_DESCRIPTION_TEMPLATE = 'Description: {description}.'
+
+
+SUPPORTED_CHECK_TYPES = (
+    'HTTP',
+    'HTTP_CUSTOM'
+    'HTTPS',
+    'SMTP',
+    'POP3',
+    'IMAP',
+    'PING',
+    'DNS',
+    'UDP',
+    'PORT_TCP',
+)
+
+
+@api_key_only_webhook_view
+@has_request_variables
+def api_pingdom_webhook(request, user_profile, stream=REQ(default='pingdom')):
+
```

Comment:
remove this newline -- we don't have newlines at the start of functions in the Zulip style.